src/peer-store/index.js

```diff
@@ -37,9 +37,11 @@ class PeerStore extends EventEmitter {
   /**
    * @constructor
    */
-  constructor () {
+  constructor ({ peerId } = {}) {
```

Comment:
Why is this optional? I don't see a use case for it and it just adds unnecessary checks when filtering self out.